templates/pages/checkout.html

```diff
@@ -1 +1,35 @@
-brand.html
+<!doctype html>
+<html>
```

Comment:
Hey @snaderiBC, I think we should try to use a layout template if possible. `empty` template should be suitable for checkout:

```html
{{#partial "head"}}

<!-- Stylesheets go here -->

{{{ stylesheet '/assets/css/optimized-checkout.css' }}}

{{/partial}}

{{#partial "page"}}

<!-- Header goes here -->

{{{ checkout_content }}}

{{/partial}}

{{> layout/empty}}
```

This way is consistent with how other pages in Cornerstone are built.